scripts/bundlesize/bundlesize.config.json

```diff
@@ -2,7 +2,7 @@
     "files": [
         {
             "path": "packages/@lwc/engine-dom/dist/index.js",
-            "maxSize": "22KB"
+            "maxSize": "22.10KB"
```

Comment:
Not super important, but I wonder if any of the suggested implementations for `ncls` would avoid the need for this size bump.

Reply:
@wjhsf This test really exists to catch unintended size increases, so this increase isn't an issue.